src/components/Molecules/molecules.story.js

```diff
@@ -99,6 +101,91 @@ storiesOf('Molecules/CardItem', module)
     />
   ));
 
+/**
+ * Add storybook definition for CtaPrompt.
+ */
+storiesOf('Molecules/CtaPrompt', module)
+  .addDecorator(checkA11y)
+  .add('Push Down', () => (
+    <CtaPrompt>
+      <CtaMessage
+        data={{
+          name: 'push_down',
+          hash: '123abcd',
+          showLogo: true,
+          title: 'Google is the #1 search engine.',
+          description: "Don't belive us? Google it.",
+          action: {
+            label: 'Google It',
+            btnColor: 'Orange',
+            url: 'https://www.google.com/'
+          },
+          dismiss: {
+            label: 'Yey, I know'
+          }
+        }}
+      />
+      <CtaMessage
+        data={{
+          name: 'push_down_2',
+          hash: '123abcd',
+          showLogo: false,
+          title: 'Bing is pretty good, too.',
+          description: 'Need to be sure? Google it.',
+          action: {
+            label: 'Google It',
+            btnColor: 'Orange',
+            url: 'https://www.google.com/'
+          },
+          dismiss: {
+            label: 'Nah, I doubt it'
+          }
+        }}
+      />
+    </CtaPrompt>
+  ));
+
+storiesOf('Molecules/CtaPrompt', module)
+  .addDecorator(checkA11y)
+  .add('Load Under', () => (
+    <CtaPrompt type="loadUnder">
+      <CtaMessage
+        data={{
+          name: 'load_under',
+          hash: '123abcd',
+          showLogo: true,
+          title: 'Google is the #1 search engine.',
+          description: "Don't belive us? Google it.",
```

Comment:
For the load under, we don't need a separate title and description. One or the other will do.

Reply:
These need to be normalized under the assumption that the description can not have HTML. The title has been restyled to appear as bold text.